### DISCUSSION TITLE
New code view cannot display external git submodule correctly

### DISCUSSION THREAD

**Discussion by RadxaYuntian** [community/community#39905] [Repositories]
### Select Topic Area

Bug

### Body

Trying to report the webpage issue and hopefully this is the correct place.

The symptom is simple. When your git submodule is linked outside of GitHub, the new code view cannot display them correctly.

You can see it from one of [our repo](https://github.com/radxa-pkg/rockchip-prebuilt/blob/main/README.md), where external submodule `debian` is displayed as `<span title="debian @ 20274a1">debian @ 20274a1</span>`.

**Comment by ossanna16** [comment]
Hi there @RadxaYuntian and welcome to our community! Thank you for reporting this bug!
